products/citrix-apps-desktops.md

```diff
@@ -15,6 +15,10 @@ releaseDateColumn: true
 sortReleasesBy: releaseDate
 LTSLabel: "<abbr title='Long Term Service Release'>LTSR</abbr>"
 releases:
+-   releaseCycle: "2209 (CR)"
+    support: 2023-03-29
+    eol: 2024-03-29
```

Comment:
According to https://www.citrix.com/support/product-lifecycle/product-matrix.html the support and eol is `2023-03-29`

Reply:
This is confusing from Citrix, as above that table they state that support is 6 months and EOL is 18 months.

That is also documented in further detail here: https://www.citrix.com/support/product-lifecycle/citrix-virtual-apps-and-desktops.html

I'm not sure if they've changed their policy but not fully updated their documentation?

Reply:
That is indeed confusing. On the product matrix page the following dates for 2209 are listed:
EOM: 29-Mar-22
EOL: 29-Mar-23

The EOM is before the release of 2209 :smile: 

I resolve this and we merge with your calculation according to the policy :+1: